app/controllers/comments_controller.rb

```diff
@@ -52,11 +52,7 @@ def edit
   # POST /comments
   # POST /comments.json
   def create
-    if RateLimitChecker.new(current_user).limit_by_action("comment_creation")
-      skip_authorization
-      render json: { error: "too many requests" }, status: :too_many_requests
-      return
-    end
+    rate_limit!(:comment_creation)
```

Comment:
should this one also be in a `before_action`, similar to [what you did with](https://github.com/thepracticaldev/dev.to/pull/7636/files#diff-8d80e37c99b30c15c5a8044a782ba1a6R3) `ImageUploadsController`?

Reply:
I would need to make another method for it since you can't pass arguments to before_actions, I tried! which is why I left it this way.